src/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -188,7 +188,6 @@ public void ResolveType(Type t)
             {
                 new object[] { 1234 },
                 new object[] { typeof(ModuleTests).GetMethod("ResolveType").MetadataToken },
-                new object[] { typeof(ModuleTests).MetadataToken + 10000 },
```

Comment:
I wrote this test. We already have 2 other bad token ids. This one caused more problems than it is worth because it causes unrelated PRs to fail if they add enough types to make this value valid.